src/interface/report/Results/PlayerInfo.tsx

```diff
@@ -88,12 +63,8 @@ const PlayerInfo = ({ combatant }: Props) => {
         <div className="player-details-talents">
           {isRetail && <PlayerInfoTalents talents={talents} />}
         </div>
-        {isRetail && (
-          <div className="player-details-talents">
-            <h3>Conduits </h3>
-            {conduits?.map((conduit) => renderConduit(conduit))}
-          </div>
-        )}
+        {isRetail && <PlayerInfoConduits conduits={conduits} />}
+        {isRetail && <PlayerInfoSoulbinds soulbinds={soulbinds} />}
```

Comment:
Is it an idea to encapsulate all 3 elements in a singular isRetail check rather then seperate check for each individually?